### ISSUE TITLE
Small UI typo

### ISSUE THREAD

**Issue by shawnyeager** [HermanMartinus/bearblog#269]
FYI. Head —> Header

![image](https://github.com/HermanMartinus/bearblog/assets/980297/1dacd476-d48a-4a7d-a307-11176ac185d3)

**Comment by HermanMartinus** [comment]
So this isn't actually a typo, but just a bit of a confusing one. In HTML you have both `<head>` and `<header>` elements that play very different roles. In this case the directive is for `<head>` where you can add meta tags, javascript, and the like. `<header>` is reserved for HTML structural elements such as titles and nav bars. 

This is a bit confusing since `<footer>` is right beneath it, but it's generally accepted to add javascript to this element. 

Long story short, I may need to give this a think and update the labels to be more clear.

**Comment by shawnyeager** [comment]
Ah, I should have caught that. Cheers.
